tests/Tests/LineSegment3d.elm

```diff
@@ -0,0 +1,35 @@
+module Tests.LineSegment3d exposing (nearestPoints)
+
+import Expect
+import LineSegment3d
+import LineSegment3d.Extra
+import Point3d
+import Test exposing (Test)
+
+
+nearestPoints : Test
+nearestPoints =
+    Test.describe "nearestPoints"
```

Comment:
Aww, what about my beautiful fuzz tests? :D I'd also add the nontrivial example unit test (we can perhaps skip the robustness ones or figure out some sane Expect.within epsilons for them)

Reply:
Sorry about losing your tests. The problem is I changed the signature of the function. It isn't computing distance anymore, it's computing nearest points. And I figured deriving distance from those points would give slightly different results and break your tests

Reply:
I don't have time to improve on this PR unfortunately. I guess if you'd like to improve on it you could create a duplicate of this PR and I'll close this one.